app/service-catalog/services.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Service Catalog services"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+    - service-catalog
+works_on:
+  - konnect
+
+description: Learn about services in Service Catalog and how to configure them.
+
+breadcrumbs:
+  - /service-catalog/
+
+related_resources:
+  - text: "Service Catalog"
+    url: /service-catalog/
+  - text: Scorecards
+    url: /service-catalog/scorecards/
+  - text: Traceable integration
+    url: /service-catalog/integrations/traceable/
+  - text: GitHub integration
+    url: /service-catalog/integrations/github/
+  - text: GitLab integration
+    url: /service-catalog/integrations/gitlab/
+  - text: SwaggerHub integration
+    url: /service-catalog/integrations/swaggerhub/
+  - text: Datadog integration
+    url: /service-catalog/integrations/datadog/
+  - text: PagerDuty integration
+    url: /service-catalog/integrations/pagerduty/
+---
+
+A Service Catalog service is a collection of one or more resources from integrations.
+
+A Service Catalog service represents the following:
+* A unit of software that is typically owned by a single team
+* Exposes one or more APIs 
+* May be dependent on other Service Catalog services (as either upstream or downstream)  
+
+Service Catalog services allow you to gain visibility into all resources in your organization, including what teams or people own them. 
+
+To create a Service Catalog service, do one of the following:
+
+{% navtabs "service" %}
+{% navtab "UI" %}
+In the {{site.konnect_short_name}} UI, click **Service Catalog** > **Services** in the sidebar. Click **New service** and configure the details about your service. Then, you can map a service to an integration's resource by clicking on the service and then select "Map resources" from the **Action** dropdown menu.
```

Comment:
```suggestion
1. In the {{site.konnect_short_name}} UI, navigate to **Service Catalog** > **Services** in the sidebar. 
1. Click **New service** and configure the details about your service. 
1. Map the service to an integration: click the service, and then select "Map resources" from the **Action** dropdown menu.
```
Should this be a list of steps, according to our UI guidelines? That will be easier for the AI bot to display.